src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -220,7 +220,9 @@ def _migrate_view_table(self, src_view: ViewToMigrate, grants: list[Grant] | Non
                 self._sql_alter_from(src_view.src, src_view.rule.as_uc_table_key, self._ws.get_workspace_id())
             )
         except DatabricksError as e:
-            logger.warning(f"Failed to migrate view {src_view.src.key} to {src_view.rule.as_uc_table_key}: {e}")
```

Comment:
Don't remove warnings. Technically, you can start warning message with a table name and parse that out of logs table